app/views/components/_navigation-sidebar.html.erb

```diff
@@ -0,0 +1,50 @@
+<%
+  related_items ||= []
+  topics ||= []
+  publishers ||= []
+  collections ||= []
+  policies ||= []
+  external_links ||= []
+  related_content = [
+    {"related_items" => related_items},
+    {"topics" => topics},
+    {"publishers" => publishers},
+    {"collections" => collections},
+    {"policies" => policies},
+    {"external_links" => external_links}
+  ]
+%>
+<% if related_content.map(&:values).flatten.any? %>
+  <aside class="app-c-navigation-sidebar" role="complementary">
+    <% related_content.each do |section| %>
+      <% section.select do |section_title, links| %>
```

Comment:
Is `section` a Hash? If so wouldn't `section.each` be more obvious? Nothing is being selected by the `select` method.